x-pack/plugins/cloud_security_posture/public/pages/configurations/configurations.tsx

```diff
@@ -50,7 +50,11 @@ export const Configurations = () => {
           path={findingsNavigation.findings_default.path}
           render={() => (
             <TrackApplicationView viewId={findingsNavigation.findings_default.id}>
-              <LatestFindingsContainer dataView={dataViewQuery.data!} />
+              <LatestFindingsContainer
+                dataView={dataViewQuery.data!}
```

Comment:
Couple of general questions:
- any downside in passing `dataViewQuery` to avoid having these three props being passed together all the time?
- we don't use context that much in our code, wdyt about the context for the dataView to avoid this multi layer prop drill?